spec/components/govuk_component/configuration/summary_list_configuration_spec.rb

```diff
@@ -25,18 +25,37 @@
         end
       end
 
-      subject do
-        render_inline(GovukComponent::SummaryListComponent.new) do |sl|
-          sl.row do |row|
-            row.key(text: "key one")
-            row.value(text: "value one")
-            row.action(text: "action one", href: "/action-one")
+      context "when visually_hidden_text is supplied" do
+        let(:visually_hidden_text) { "visually hidden info" }
+        subject! do
+          render_inline(GovukComponent::SummaryListComponent.new) do |sl|
+            sl.row do |row|
+              row.key(text: "key one")
+              row.value(text: "value one")
+              row.action(text: "action one", href: "/action-one", visually_hidden_text: visually_hidden_text)
+            end
           end
         end
+
+        specify "raises an error when no visually hidden text is supplied" do
```

Comment:
```suggestion
        specify "renders a span with the visually hidden text" do
```

Reply:
Thanks @cpjmcquillan - rushing and failing 😩